website-1.x/blog/2021-03-09-releasing-docusaurus-i18n.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Releasing Docusaurus i18n
+author: Sébastien Lorber
+authorTitle: Docusaurus maintainer
+authorURL: https://sebastienlorber.com
+authorImageURL: https://github.com/slorber.png
+authorTwitter: sebastienlorber
+tags: [release, i18n]
+image: /img/blog/2021-03-09-releasing-docusaurus-i18n/social-card.png
+---
+
+Today, we officially release **[Docusaurus 2 i18n](https://v2.docusaurus.io/docs/i18n/introduction)** ([internationalization](https://en.wikipedia.org/wiki/Internationalization_and_localization)), as part of [2.0.0-alpha.71](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-alpha.71).
+
+Docusaurus 2 has now reached **full feature parity with Docusaurus 1**, and a first **beta** is on its way.
```

Comment:
What is keeping us from making this the beta release? I imagine many reading this might have the same question in their mind. If we are at full feature parity, wouldn't that constitute a beta? Or are you wanting to have i18n out in public a bit and tested before officially calling v2 a beta. If so, might be worth mentioning that.

Reply:
fixed, as seen on Workplace